docs/topics/api/auth_internal.rst

```diff
@@ -10,34 +10,31 @@ are looking for how to authenticate with the API from an external client, using
 your API keys, read the :ref:`documentation for external authentication
 <api-auth>` instead.
 
-When using this authentication mechanism, the server is responsible for
-creating an API Token when the user logs in, and sends it back in
-the response. The clients must then include that token as an ``Authorization``
-header on requests that need authentication. The clients never generate JWTs
-themselves.
+When using this authentication mechanism, the server creates a session and stores the
+session id in the ``sessionid` cookie when the user logs in.
+The client must then include that session id in an ``Authorization`` header on requests
+that need authentication.
+The clients never generate tokens or sessions themselves.
```

Comment:
Why do you mention "tokens" here when they aren't mentioned anywhere else? (Or maybe they are and I just don't see it in this diff)

Reply:
External auth uses JWTs, but perhaps the sentence is redundant anyway? (the previous version said something similar)